Ginger/Ginger/SolutionWindows/AddSolutionPage.xaml

```diff
@@ -34,7 +34,16 @@
                 <Label Style="{StaticResource @InputFieldLabelStyle}" Content="Main Application Platform:" />
                 <ComboBox x:Name="MainPlatformComboBox" Style="{StaticResource @InputComboBoxStyle}" Width="200" HorizontalAlignment="Left" />
             </StackPanel>
-                     
+            <Label Style="{StaticResource @InputFieldLabelStyle}">Encryption Key:</Label>
```

Comment:
As discussed, let's create user control for encryption key related controls. So that it can be reused in all places

Reply:
Added seperate user control for encryption key